apollo-router/src/router_factory.rs

```diff
@@ -675,10 +696,12 @@ pub(crate) async fn create_plugins(
             }
         }
     }
-    add_mandatory_apollo_plugin!("limits");
     add_mandatory_apollo_plugin!("license_enforcement");
     add_mandatory_apollo_plugin!("health_check");
+    // TODO: Introduce a CORS plugin here
```

Comment:
This will be a follow-up, just recording our decision on where to put it as a code comment